main.go

```diff
@@ -54,13 +55,17 @@ func HandleConfigFile(filePath string) (config.ConfigReader, error) {
 
 	c := viper.New()
 
+	// map settings such as foo.bar to FOO_BAR environment keys
+	c.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
+
 	//Allow AWS URI endpoint to be set by environment variable
 	c.BindEnv(SettingAwsURI, "AWS_URI")
 
 	//Allow aws keyid, aws token, aws secret to be read by viper
 	c.BindEnv(SettingAwsAuthKeyId, "AWS_ACCESS_KEY_ID")
 	c.BindEnv(SettingAwsAuthSecret, "AWS_SECRET_ACCESS_KEY")
 	c.BindEnv(SettingAwsAuthToken, "AWS_SESSION_TOKEN")
+	c.BindEnv(SettingAwsS3Region, "AWS_S3_REGION")
```

Comment:
What I'm after is a singe, unified, consistent way to deal with settings. Having a simple doc mention: `if you want to set any available configuration option though ENV just set env var with servciename_settingname variable, all CAPS, while accessing nested settings use _ or / whatever to indicate nesting, for available options look at config.yaml for specific service` and we just documented clearly how this use it.